test/shallowbackup.js

```diff
@@ -0,0 +1,140 @@
+// Copyright © 2017 IBM Corp. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* global describe it */
+'use strict';
+
+const assert = require('assert');
+const backup = require('../includes/shallowbackup.js');
+const fs = require('fs');
+const nock = require('nock');
+
+describe('#unit Perform backup using shallow back', function() {
+  const dbUrl = 'http://localhost:5984/animaldb';
+
```

Comment:
I like to see a `beforeEach` that does `nock.cleanAll()` I found it very helpful when faced with multiple mock failures in a single suite.